rust/sled/sled.go

```diff
@@ -0,0 +1,35 @@
+package sled
+
+import "github.com/goplus/llgo/c"
+
+// Write the .pc file for the dylib generated by the Rust library and copy it to pkg-config for proper location.
+const (
+	LLGoPackage = "link: $(pkg-config --libs sled); -lsled"
+)
+
+type SledConfig struct {
+	Unused [8]byte
```

Comment:
should have detail members (to get correct struct size) since we call `conf := &sled.SledConfig{}` in demo.

Reply:
In my latest [update](https://github.com/hackerchai/llgo/pull/2), the configuration is created using sled_create_config. The current status of the PR seems to have reverted this commit due to a rebase operation.